src/chains/ethereum/ethereum/package.json

```diff
@@ -70,14 +70,15 @@
     "env-paths": "2.2.1",
     "eth-sig-util": "2.5.3",
     "ethereumjs-abi": "0.6.8",
-    "ethereumjs-util": "7.1.3",
+    "ethereumjs-util": "7.1.4",
     "keccak": "3.0.1",
+    "encoding-down": "7.1.0",
```

Comment:
We were already using encoding-down directly, but were able to  import it only because it was present within a transitive dependency.